src/backgroundQueue/sinks/memberBundleRequest.sink.spec.ts

```diff
@@ -144,7 +146,7 @@ describe('memberBundleIssuance', () => {
         expect(mockPostToAwala).toHaveBeenCalledOnceWith(
           expect.anything(),
           expect.anything(),
-          new URL(envVars.AWALA_MIDDLEWARE_ENDPOINT!),
+          new URL(REQUIRED_ENV_VARS.AWALA_MIDDLEWARE_ENDPOINT),
```

Comment:
We don't need the whole `REQUIRED_ENV_VARS` and for consistency with other tests we should simply do:

```suggestion
          new URL(AWALA_MIDDLEWARE_ENDPOINT),
```